<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageWindow</class>
 <widget class="QWidget" name="MessageWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="MainLayout">
     <item row="1" column="0">
      <widget class="QLineEdit" name="MsgEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="SendMsgBtn">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Send msg</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QTableWidget" name="tableWidget">
       <property name="columnCount">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
